scripts/generate-qr.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+import fs from 'fs';
+import QRCode from 'qrcode';
```

Comment:
Do we need to install the dependency somehow?

If it is a script that is rarely used, we could put it in a separate folder with a package.json, or comment at least on the top, what external dependencies need to be installed.